app/scripts/components/sandbox/mdxEditor/index.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+
+import useLocalStorage from "use-local-storage";
+import MDXEditor, { MDXBlockWithError } from './MDX';
+import { PageMainContent } from '$styles/page';
+import ContentBlockFigure from '$components/common/blocks/figure';
+import { ContentBlockProse } from '$styles/content-block';
+import Image, { Caption } from '$components/common/blocks/images';
+import { Chapter } from '$components/common/blocks/scrollytelling/chapter';
+import { NotebookConnectCalloutBlock } from '$components/common/notebook-connect';
+import { LazyMap, LazyScrollyTelling, LazyChart, LazyCompareImage} from '$components/common/blocks/lazy-components';
+import SmartLink from '$components/common/smart-link';
+
+
+const components = {
+  h1: (props) => <h1 style={{ color: 'tomato' }} {...props} />,
+  Test: () => <div>Test</div>,
+  Block: MDXBlockWithError,
+  Prose: ContentBlockProse,
+  Figure: ContentBlockFigure,
+  Caption,
+  Chapter,
+  Image,
+  Map: LazyMap,
+  ScrollytellingBlock: LazyScrollyTelling,
+  Chart: LazyChart,
+  CompareImage: LazyCompareImage,
+  NotebookConnectCallout: NotebookConnectCalloutBlock,
+  Link: SmartLink
+};
+
+export const MDX_LOCAL_STORAGE_KEY = "MDX_EDITOR";
```

Comment:
nitpick: It might be a good idea to start prefixing these local storage keys with `VEDA` or something?

Reply:
Maybe you're right... Maybe it should be part of a broader effort to centralize local storage handling within the app? I created an issue for now: https://github.com/NASA-IMPACT/veda-ui/issues/564